docs/user-guide/lazy-loading.md

```diff
@@ -0,0 +1,91 @@
+# Lazy Loading
+
+> [!WARNING]
+> This is an experimental option and may not work as expected with all plugins.
+> The API may change without notice. > Please report any issues you encounter.
```

Comment:
```suggestion
> The API may change without notice.
>
> Please report any issues you encounter.
```
or
```suggestion
> The API may change without notice. \
> Please report any issues you encounter.
```

Was this indented?

Reply:
![image](https://github.com/user-attachments/assets/a49d2953-5dab-4f38-bcd9-115660416774)

Reply:
Hmm my conform kept moving it up, for some reason.

Reply:
Alright, fixed it.
![image](https://github.com/user-attachments/assets/a20960e6-db2e-4300-9070-198b3dda76c7)